### ISSUE TITLE
[Bug] iOS - ListView cell height

### ISSUE THREAD

**Issue by MarMarIV** [xamarin/Xamarin.Forms#6638]
### Description
Hi, i have a problem with cell height on iOS with using DataTemplateSelector on ListView. Sometimes happend, the cell height is not calculated exactly (in screenshot)

### Steps to Reproduce

1. DataTemplateSelector for use diferent layout

### Basic Information

- Version with issue: 3.4.0.1029999
- IDE: VS2019 16.1.2 
- Platform Target Frameworks: 
  - iOS:  12.2
- Affected Devices:
  iPhone SE

### Screenshots

![8104fd3b-c4af-4cc8-bd04-963779459d9a](https://user-images.githubusercontent.com/10964247/59990836-132ae980-9645-11e9-95e3-196dd02c87f6.png)

**Comment by KSemenenko** [comment]
I have the same bug too.

**Comment by samhouts** [comment]
@MarMarIV @KSemenenko  Can you please attach a small project that demonstrates this issue? Thanks!

**Comment by KSemenenko** [comment]
I'm already working on a project that will show this bug

**Comment by KSemenenko** [comment]
@MarMarIV Maybe you can also make a test project?

**Comment by samhouts** [comment]
Thanks! We'll look for it.

**Comment by MarMarIV** [comment]
I try to reprodece it on empty project (+ ListView wit DateTemplateSelector), but i cannot

**Comment by KSemenenko** [comment]
@MarMarIV I have not finished my test, but I think this is due to the fact that there are several DataTemplate of different heights and Entry or Editr inside.

**Comment by samhouts** [comment]
Any luck with a reproduction? Thanks!

**Comment by KSemenenko** [comment]
Sorry for so long, here is an example.
The problem is that when values change through binding, the cell size does not change. Also It can be related to #5115
[GroupHeaderTemplateLayoutErroriOS.zip](https://github.com/xamarin/Xamarin.Forms/files/3446940/GroupHeaderTemplateLayoutErroriOS.zip)
@samhouts

**Comment by KSemenenko** [comment]
This seems to require calling ForceUpdateSize.
Or InvalidateMeasure after changing the BindingContext.

**Comment by giuseppenovielli** [comment]
Yes, ListView with DataTemplateSelector and also Grouped Height the cell height is not calculated exactly. 
There are any Workaround?
